how-to/register-with-browser/client/src/platform-override.ts

```diff
@@ -0,0 +1,167 @@
+import {
+	AnalyticsEvent,
+	CreateSavedPageRequest,
+	CreateSavedWorkspaceRequest,
+	getCurrentSync,
+	GlobalContextMenuItemTemplate,
+	GlobalContextMenuOptionType,
+	OpenGlobalContextMenuPayload,
+	OpenPageTabContextMenuPayload,
+	OpenViewTabContextMenuPayload,
+	Page,
+	UpdateSavedPageRequest,
+	UpdateSavedWorkspaceRequest,
+	Workspace,
+	WorkspacePlatformOverrideCallback
+} from "@openfin/workspace-platform";
+
+export const overrideCallback: WorkspacePlatformOverrideCallback = async (WorkspacePlatformProvider) => {
+	class Override extends WorkspacePlatformProvider {
+		public async getSnapshot(...args: [undefined, OpenFin.ClientIdentity]) {
+			return super.getSnapshot(...args);
+		}
+
+		public async applySnapshot(...args: [OpenFin.ApplySnapshotPayload, OpenFin.ClientIdentity]) {
```

Comment:
Do we really need all the empty overrides, it just adds cruft to what we are actually trying to demonstrate, customize has all this. These more targeted samples need to be more succinct.

Reply:
Yep this needs the overrides. I know it adds a lot of noise but people asking how to I override save workspace or page for my platform feel overwhelmed if we point them to customize (as it does so much more). This is a very basic example of here are the hooks.